src/thriveopedia/pages/ThriveopediaMuseumPage.tscn

```diff
@@ -10,6 +10,7 @@
 [ext_resource path="res://src/gui_common/dialogs/CustomConfirmationDialog.tscn" type="PackedScene" id=8]
 
 [node name="ThriveopediaMuseumPage" instance=ExtResource( 1 )]
+rect_pivot_offset = Vector2( -469, -108 )
```

Comment:
This property needs to be unset (unless there's really something that must have this to function properly).